components/templates/CookieBanner.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div
+    class="
+      bg-white
+      dark:bg-sky-black
+      border-t border-t-sky-light
+      text-center
+      md:text-left
+      text-sky-dark
+      dark:text-white
+      font-medium
+    "
+  >
+    <VueIfBot>
+      <CookieConsent
+        class="
+          d-container-content
+          py-4
+          px-4
+          sm:px-6
+          lg:px-8
+          flex flex-col
+          md:flex-row
+          items-center
+          md:justify-between
+          space-y-4
+          md:space-y-0
+        "
+      >
+        <template slot="message">
+          <p class="md:w-1/2">
+            {{ $t('cookies.message') }}
+            <a class="underline" rel="noopener" target="_blank" href="https://www.cookiesandyou.com/">
+              {{ $t('cookies.link') }}
+            </a>
+          </p>
+        </template>
+        <template slot="button">
+          <SectionButton
+            size="md"
+            class="bg-primary text-black hover:bg-primary-400 focus:bg-primary-400"
+            @click.native="$emit('cookie-banner')"
+          >
+            {{ $t('cookies.button') }}
+          </SectionButton>
+        </template>
+      </CookieConsent>
+    </VueIfBot>
+  </div>
+</template>
+
+<script>
+import VueIfBot from 'vue-if-bot/dist/vue-if-bot.es'
+import CookieConsent from 'vue-cookieconsent-component/src/components/CookieConsent.vue'
```

Comment:
According to the docs it does not seem right to import them this way. Is there a reason?
https://github.com/Developmint/vue-if-bot#synchronous-import
https://github.com/EvodiaAut/vue-cookieconsent-component#usage

Reply:
I think I have to update the docs 🙈 
`vue-if-bot` import seems alright 👍 
https://github.com/manniL/lichter.io/blob/31cded247d5c2cad2bb6f29be5355e46bec9ffdb/layouts/default.vue#L18